cmd/documentation/exchanges_templates/subscription.tmpl

```diff
@@ -0,0 +1,38 @@
+{{define "exchanges subscription" -}}
+{{template "header" .}}
+# Exchange Subscriptions
+
+Exchange Subscriptions are streams of data delivered via websocket.
+
+GoCryptoTrader engine will Subscribe automatically to configured channels.
+A subset of exchanges currently support user configured channels, with the remaining using hardcoded defaults.
+See configuration Features.Subscriptions for whether an exchange is configurable.
+
+## Templating
+
+Exchange Contributors should implement `GetSubscriptionTemplate` to return a text/template Template.
+
+Exchanges are free to implement template caching, a map or a mono-template, inline or file templates.
+
+The template is provided with a single context structure:
+```
```

Comment:
Just adds some lovely highlighting
```suggestion
```go
```

<img width="330" alt="image" src="https://github.com/thrasher-corp/gocryptotrader/assets/9391715/ce930794-571d-4fc5-94e3-e78bdf745355">

Reply:
Yep. Thought about that, so 👍 

Fixed 30803b3